projects/plugins/jetpack/extensions/blocks/videopress/v6/edit.js

```diff
@@ -206,14 +208,32 @@ export default function VideoPressEdit( { attributes, setAttributes } ) {
 		onSuccess: setAttributes,
 	} );
 
+	const getGuidFromVideoUrl = url => {
+		try {
+			const urlObject = new URL( url );
+			if ( urlObject.protocol === 'https:' && urlObject.host === 'videopress.com' ) {
```

Comment:
Private videos use `video.wordpress.com` as well so that should be added to this if statement I think

Reply:
Addressing here https://github.com/Automattic/jetpack/pull/24918